sdk/core/azure-core-tracing-opentelemetry/CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 -------------------
 ## 1.0.0b6 (Unreleased)
 
+- Pinned opentelemetry-api to version 0.10b0
```

Comment:
I thought you told it got released as stable?

Reply:
yes - my bad  - checked github did not have pre release tag - but i was wrog 😁